_component_design/buttons.md

```diff
@@ -233,32 +191,18 @@ Compact buttons should be used when you have severe space constraints, such as i
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Button sets
-
-- Buttons should be aligned in order of primary, secondary, and tertiary.
-- Button sets are most often used in Popovers, Modals or Create/Edit Forms to group the submit, back or next, and cancel buttons.
-- A button set should never contain more than one primary button.
-- Avoid using split buttons in a button set as it would complicate what should be a simple decision.
-- The cancel action must always be an `hxBtn.hxLink` as shown in the example.
+## Primary button
 
-When submitting forms:
+A primary button indicates the primary action on a page. Use only one primary button per page.
```

Comment:
A primary button indicates the primary action a user can take. 

Primary buttons are designed to command the attention of users over and above any other content on the screen. 

Do not use multiple primary buttons in the same view. This causes visual confusion and increases cognitive load while decreasing the efficacy of a primary button to direct user attention on a single action.  

For example, do not use primary buttons in a dashboard widget because it will attract too much user attention and their eyes cannot settle and focus on a single widget.